### PR TITLE
Add test for getImageArray, getTextArray in origami library!

### PR DESCRIPTION
**Pull request by ccr4b**

### CODE REVIEW

File: origami-lib/test/test_textimageinput.py [hunk 1]
```diff
@@ -28,6 +28,20 @@ def test_getTextInput():
     assert(len(text_array) == 3)
     assert(text_array == ['hello', 'world', 'Is this a sample question?'])
 
+# send text strings
+def test_getTextInput2():
```
Comment (deshraj) [inline origami-lib/test/test_textimageinput.py]: Thanks for writing this but this seems to be same as the previous test case. Can you please try writing tests for the corner cases. For example, if the inputs are empty then what will be output and so on.

File: origami-lib/test/test_textimageinput.py [hunk 2]
```diff
@@ -59,6 +73,17 @@ def test_getImageInput_filepath():
     for i in range(len(filepaths)):
         assert(np.array_equal(cv2.imread(filepaths[i]), cv2.imread(image_list[i])))
 
+# send image and get result as file path
+def test_getImageInput_filepath2():
```
Comment (deshraj) [inline origami-lib/test/test_textimageinput.py]: This test seems to be a part of the previous test. So please remove this.

**Comment by AvaisP** [comment]
Please fix the build first
